port/unix/omrsock.c

```diff
@@ -550,7 +550,19 @@ omrsock_accept(struct OMRPortLibrary *portLibrary, omrsock_socket_t serverSock,
 int32_t
 omrsock_send(struct OMRPortLibrary *portLibrary, omrsock_socket_t sock, uint8_t *buf, int32_t nbyte, int32_t flags)
 {
-	return OMRPORT_ERROR_NOT_SUPPORTED_ON_THIS_PLATFORM;
+	int32_t bytesSent = 0;
+
+	if (NULL == sock || 0 >= nbyte) {
+		return OMRPORT_ERROR_INVALID_ARGUMENTS;
+	}
+
+	bytesSent = send(sock->data, buf, nbyte, flags);
+	if (-1 == bytesSent) {
+		portLibrary->error_set_last_error(portLibrary, errno, OMRPORT_ERROR_SOCK_SEND_FAILED);
+		return -1;
```

Comment:
You won't need the return -1 here, you can fall through because bytesSent is -1.

Reply:
Oh I see what you mean, I will change that :+1: